captum/attr/_core/neuron/neuron_feature_ablation.py

```diff
@@ -33,13 +42,15 @@ def __init__(self, forward_func, layer, device_ids=None):
 
     def attribute(
         self,
-        inputs,
-        neuron_index,
-        baselines=None,
-        additional_forward_args=None,
-        feature_mask=None,
-        attribute_to_neuron_input=False,
-        ablations_per_eval=1,
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
```

Comment:
You can actually use the TensorOrTupleOfTensors type here, and then the return type would be the same (this conveys that the return type matches the inputs type). That type can be imported from captum.attr.utils.typing.